src/components/gas/GasSpeedButton.android.js

```diff
@@ -380,7 +380,7 @@ const GasSpeedButton = ({
                   onChangeText={handleCustomGasChange}
                   onFocus={handleCustomGasFocus}
                   onSubmitEditing={handleInputButtonManager}
-                  placeholder={`${defaultCustomGasPrice}`}
+                  placeholder={`${defaultCustomGasPrice}${android ? ' ' : ''}`}
```

Comment:
It's .android file so no need to make a code platform-specific.
Also, can you add maybe some comment?

Reply:
```suggestion
                  placeholder={`${defaultCustomGasPrice} `}
```